### PR TITLE
BZ2002582: Rewording supported cluster section for updated OCP version support

### PR DESCRIPTION
**Pull request by mgarrell777**
This PR addresses (https://bugzilla.redhat.com/show_bug.cgi?id=2002582) and its associated JIRA story (https://issues.redhat.com/browse/CNV-14113).

The bug is about modifying the OpenShift Virtualization Supported Cluster Version section to indicate that the latest OCP version is needed.

CP: 4.7-4.10

Preview: https://deploy-preview-38206--osdocs.netlify.app/openshift-enterprise/latest/virt/about-virt#virt-supported-cluster-version_about-virt

NOTE: The preview build is based on main. So the version number will show as the most recent, rather than the older version specified in the bug. However, the version number that will show in the final build will be the correct, cherrypicked branch.

### CODE REVIEW

**Review by dominikholler — CHANGES_REQUESTED** [review]

**Review by dominikholler — APPROVED** [review]
Thanks, looks good.

**Comment by dbasunag** [comment]
lgtm

File: modules/virt-supported-cluster-version.adoc [hunk 1]
```diff
@@ -6,5 +6,4 @@
 [id="virt-supported-cluster-version_{context}"]
 = {VirtProductName} supported cluster version
 
-{VirtProductName} {VirtVersion} is supported for use on {product-title} {product-version} clusters.
-
+{VirtProductName} {VirtVersion} is supported for use on {product-title} {product-version} clusters. To use the latest z-stream release of {VirtProductName}, you must have updated your cluster to the latest version of {product-title} {product-version}.
```
Comment (sjhala-ccs) [inline modules/virt-supported-cluster-version.adoc:9]: You don't need the {product-version} variable at the end of the second sentence. 

I would rewrite the second sentence as: "To use the latest z-stream release of {VirtProductName}, you must first upgrade to the latest version of {product-title}."

**Review by sjhala-ccs — COMMENTED** [review]
Just one nit and a suggestion for you to consider, thanks Matthew!

**Pull request merged**